.github/workflows/build-verification.yml

```diff
@@ -16,7 +15,7 @@ jobs:
       - id: setup-matrix
         run: echo "matrix=$(jq -cM '.supportedVersions | keys' src/main/resources/versions.json | sed -e 's/\./_/g' -e 's/-/_/g')" >> $GITHUB_OUTPUT
       - name: debug
-        run: echo ${{ steps.setup-matrix.outputs.matrix }}
+        run: echo "matrix=${{ steps.setup-matrix.outputs.matrix }}"
```

Comment:
just curious, why this change?

Reply:
Oh, it's still left from the first variant where I set the PTS mode in that job, but I still left it in as it's more verbose when looking at logs. I can remove this change if wanted.

Reply:
let's leave it how it originally was to simplify and reduce the changeset.